splash-screen/android/src/main/java/com/capacitorjs/plugins/splashscreen/SplashScreen.java

```diff
@@ -571,6 +572,16 @@ private void hideDialog(final AppCompatActivity activity, boolean isLaunchSplash
             return;
         }
 
+        // Hide with Android 12 API
+        if (null != this.onPreDrawListener) {
+            this.isVisible = false;
+            this.onPreDrawListener = null;
```

Comment:
same as the other comment, `this.onPreDrawListener = null;` should be after `removeOnPreDrawListener` or won't have any effect.